packages/mui-joy/src/Slider/Slider.tsx

```diff
@@ -476,12 +476,14 @@ Slider.propTypes /* remove-proptypes */ = {
     thumb: PropTypes.object,
     track: PropTypes.object,
     valueLabel: PropTypes.shape({
+      children: PropTypes.element,
       className: PropTypes.string,
       components: PropTypes.shape({
         Root: PropTypes.elementType,
       }),
+      open: PropTypes.bool,
       style: PropTypes.object,
-      value: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.number]),
+      value: PropTypes.number,
```

Comment:
I don't know why, but proptypes check failed on Slider https://app.circleci.com/pipelines/github/mui/material-ui/74334/workflows/e190eb76-5d4e-4c71-bb6a-71edf188a337/jobs/394411 so I had to commit those changes for CI to pass

Reply:
No worry. I guess it might be from a stale PR that is approved but does not pull the latest master.